pkg/api/resourcegroup/v1alpha1/types.go

```diff
@@ -0,0 +1,47 @@
+// Copyright 2021 Google LLC
```

Comment:
can you rebase this PR since this code has been merged?

Reply:
Oddly, this PR is rebased to `main`, but this change is still showing up...